packages/react/src/components/UIShell/components/Sharkfin.tsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { usePrefix } from '../internal/usePrefix';
+
+export interface SharkfinIconProps {
+  /**
+   * Provide a single icon as the child to `SharkfinIcon` to render in the
+   * container
+   */
+  children?: React.ReactNode;
+
+  /**
+   * Provide an optional class to be applied to the containing node
+   */
+  className?: string;
+
+  /**
+   * Specify whether the icon should be placed in a smaller bounding box
+   * Since the 'small' prop is not provided, we make it optional and set a default value to `false`.
+   */
+  small?: boolean;
+}
+
+export const SharkfinIcon: React.FC<SharkfinIconProps> = ({
+  children,
+  className: customClassName,
+  small = false,
+}) => {
+  const prefix = usePrefix();
+  const className = cx({
+    [`${prefix}--side-nav__icon`]: true,
```

Comment:
We should probably name this classname the same as the component, and split these styles into their own style sheet. I could see this being completely separate from the sidenav once we bring it to core. Let's also make the file name consistent with the component name.